### ISSUE TITLE
The hamburger button cannot be keyboard focussed

### ISSUE THREAD

**Issue by OllyHodgson** [alexanderlperez/headergrabbag#2]
When tabbing through the links in the page, it quickly becomes clear that you're unable to focus or activate the hamburger menu without using a mouse / finger.

There's a few ways to fix this:

1. Replace `<div class="mobile-toggle-wrap">` with an `<a>` element, setting the `href` to the `id` of the menu container. This way it can still work in a non-JS scenario.
2. Wrap the button in a `<button>` element, instead of a `<div>`
3. Give the `<div class="mobile-toggle-wrap">` element a `role="button"` attribute, and a `tabindex` attribute

In all cases, you should also add an `aria-label` attribute to that element, e.g. `aria-label="Toggle navigation"`: This allows users of assistive technology (e.g. a blind or partially sighted user with a screen reader) to be told what the button does. Right now they'll probably be told there's a list with three empty list items. ([Bootstrap's navbar component](http://getbootstrap.com/components/#navbar) seems to be a pretty good example of how to do this stuff properly.)

**Comment by alexanderlperez** [comment]
Ooo great catch! I'm a Vimium user myself (at least 60.001% of the time) and lo and behold it won't work without it something like `role="button"`, which is what I've put in along with the `tabindex`, along w/ `aria-label`.
